compose.yml

```diff
@@ -1,7 +1,10 @@
 version: '1'
 services:
-  # neo4j-store:
-  #   build: "./data/"
+  neo4j-dbms:
+    build: "data/"
+    ports:
+      - 7474:7474
+      - 7687:7687
```

Comment:
I already have a dockerize neo4j running which conflicts with these ports, perhaps we should use non-standard ports within this compose file

Reply:
This will require changing the image which from some initial research looks like a lot of work. I'd say it's better for us (at least for now) to just stop the databases we're running locally so we can use the pure image for neo4j